extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -158,6 +159,9 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     this.boundNoContentHandler_ = () => this.noContentHandler_();
 
     setupA2AListener(this.win);
+
+    /** @private {string|undefined} */
+    this.container_ = undefined;
```

Comment:
We should always be setting properties in the `#buildCallback` to get a consistent hidden class.